### ISSUE TITLE
Visual focus is lost after changes maded in the Host drop-down

### ISSUE THREAD

**Issue by zqian** [tl-its-umich-edu/remote-office-hours-queue#752]
Host combo box: After changing the combo box option using keyboard or mouse, the visual focus is lost. Need to hit Tab again to resume. This appears to be an accessibility fail caused by a JavaScript listener issue.

<img width="828" height="134" alt="Image" src="https://github.com/user-attachments/assets/fb0f845f-37ea-406c-9c01-2589c9e4304d" />

**Comment by lsloan** [comment]
I expect a possible solution may be to add code to the JS listener function which reassigns focus to the menu before the function exits.
